luigi_pipeline/lib/hail_tasks.py

```diff
@@ -203,7 +203,7 @@ class HailElasticSearchTask(luigi.Task):
     es_index = luigi.Parameter(description='ElasticSearch index.', default='data')
     es_username = luigi.Parameter(description='ElasticSearch username.', default='pipeline')
     es_password = luigi.Parameter(description='ElasticSearch password.', visibility=ParameterVisibility.PRIVATE, default=None)
-    es_index_min_num_shards = luigi.IntParameter(default=6,
+    es_index_min_num_shards = luigi.IntParameter(default=1,
```

Comment:
this parameter looks like it's used on line 246 (https://github.com/broadinstitute/hail-elasticsearch-pipelines/blob/master/luigi_pipeline/lib/hail_tasks.py#L246) to determine whether this int or the calculated int is larger. would it make sense to make the `max` on line 246 a `min`? or should we just always set the number of shards equal to the calculated number. I can't imagine this number will ever be larger than the calculated number now. for example, I checked this project MT: `gs://seqr-datasets/v02/GRCh38/RDG_WGS_Broad_Internal/v18/projects/R0312_cmg_bonnemann_genomes.mt`, and this MT would have 3 shards based on `_mt_num_shards`

Reply:
I'm not sure I understand how min would work. The max is there because we hit limits with the number of docs within an index so we should always take the max number.

Reply:
ah gotcha. the max isn't required anymore then is it? would any of our project MTs calculate to have a fewer than 1 shard? line 246 will probably always return `calculated_num_shards` now, right?

Reply:
Yeah,  the calculation is always rounded to the next integer so 1 is the smallest -- max is not needed unless for some reason we would want to spread the information over more shards than the calculation? I can't imagine a situation when we'd want that considering this PR

Reply:
so to clarify, changing this default and leaving the rest of the code as-is has the desired effect, correct?

Reply:
Yes it does